Umbraco.gitignore

```diff
@@ -19,6 +19,10 @@
 ## Uncomment this line if you think it fits the way you work on your project.
 ## **/[Uu]mbraco/ 
 
+## This [Mm]edia/ folder contains content. Content may vary by environment and should therefore not be added to source control.
```

Comment:
I might say `The` rather than `This`, but otherwise LGTM!
```suggestion
## The [Mm]edia/ folder contains content. Content may vary by environment and should therefore not be added to source control.
```

Reply:
@bl-ue  That was blazing fast! Thanks for your suggestion.